NewForce-Cohort-9/tabloid-full-stack-code-crusaders
# Edit & Delete Post - ticket7

### Description
Created Edit Post page with a form to edit existing posts.

Made the option to DELETE a post from the post page. Form provides the ability to delete an existing post in the database and update the displayed list on the Post List page. (needs perfecting)

### Type of change

 New feature (non-breaking change which adds functionality)

### Testing Instructions

1. git add and git commit whatever you're working on on your branch
2. git checkout main
3. git fetch --a
4. git checkout ml-DeletePost-ticket7
5. If you don't see the most recent version of my files, git pull origin ml-DeletePost-ticket7
6. If you haven't created the Tabloid MVC database, please do so:

- Use this to fill out tables and columns: [SQL data](https://github.com/NewForce-Cohort-9/newforce-c9-tabloid-full-stack-TabloidFullStack/blob/main/SQL/01_Tabloid_Create_DB.sql)
- Use this to seed the database: [SQL Seed Data](https://github.com/NewForce-Cohort-9/newforce-c9-tabloid-full-stack-TabloidFullStack/blob/main/SQL/02_Tabloid_Seed_Data.sql)

7. Click the execute with debugger solid green button to launch the app
8. Run npm start from the client/tabloid directory in the terminal
9. In browser, please go to Login on the nav bar and log in with this email: [[foo@bar.com](mailto:foo@bar.com)] and random password (doesn't matter)
10. On the home page - Click the Posts menu option
11. When they select the Posts link in the menu, they should be directed to a page that lists all the Posts in order of publication date with the most recent on top
12. Create a new post "A New Post". After creation, you will be redirected to the Post List page
13. Edit the post "A New Post", change the name to "A New Post Edited". After editing, you will be redirected to the Post List page.
14. Try to Delete the post "A New Post Edited", this will direct you to a page asking you to confirm if you want to delete the post, Click confirm delete 

### Checklist:
